static/js/vue-cdr-access/src/utils/helper_methods.js

```diff
@@ -0,0 +1,55 @@
+const utils = {
+    /**
+     * Put URL paramaters into an object
+     * @returns {{start: number, page: number, sort: string, rows: number}}
+     */
+    urlParams() {
```

Comment:
Since it appears that we have access to the $route, I wonder if instead of processing the url ourselves we could use `$route.params` or `$route.query`? They both appear to return a hash. I'm just skimming here though so might not know what i'm talking about

https://router.vuejs.org/api/#route-object-properties